src/styles/s3.css

```diff
@@ -0,0 +1,101 @@
+
+/**
+ * @author Jamie jamie.h@inwinstack.com
+ */
+ 
+/* border radius ----------------------------------------- */
+.radius-4px {
```

Comment:
The class name should not be named as property value. It should be meaningful, ex. rounded-small.